frontend/lib/tests/mock-fetch.ts

```diff
@@ -27,7 +27,7 @@ export function createMockFetch(): MockFetch {
   window.fetch = mock;
   return Object.assign(mock, {
     mockReturnJson(this: jest.Mock, result: any) {
-      this.mockResolvedValue({ json: () => Promise.resolve(result) });
+      this.mockResolvedValue({ status: 200, json: () => Promise.resolve(result) });
```

Comment:
Had to do this because `@justfixnyc/geosearch-requester` contains a bugfix that ensures the status code of the response is 200 (if it's not, it treats the request as a network error).